redbot/core/core_commands.py

```diff
@@ -1359,6 +1396,42 @@ async def regionalformat(self, ctx: commands.Context, language_code: str = None)
         standardized_locale_name = f"{locale.language}-{locale.territory}"
         i18n.set_regional_format(standardized_locale_name)
         await ctx.bot._config.regional_format.set(standardized_locale_name)
+        await ctx.send(
+            _("Global regional formatting will now be based on `{language_code}` locale.").format(
+                language_code=standardized_locale_name
+            )
+        )
+
+    @_set.command(aliases=["region"])
+    @checks.guildowner_or_permissions(manage_guild=True)
+    async def regionalformat(self, ctx: commands.Context, language_code: str = None):
```

Comment:
This needs `@commands.guild_only()`

Reply:
Not really necessary 

```py
@checks.guildowner_or_permissions(manage_guild=True
```
Will account for that

Reply:
AAAA As Jack points out ..... Although this is the case bot everyone thats not a bot owner ... this would still fail for bot owners 🤦 

So yes `@commands.guild_only()` would be needed here to avoid bot owners running this in DMs

Reply:
> AAAA As Jack points out ..... Although this is the case forr everyone thats not a bot owner ... this would still fail for bot owners facepalm
> 
> So yes `@commands.guild_only()` would be needed here to avoid bot owners running this in DMs

That would be why it was in my DMs

Reply:
Added in a pending commit.